Gopkg.lock

```diff
@@ -313,7 +303,8 @@
 [[projects]]
   name = "github.com/jinzhu/gorm"
   packages = ["."]
-  revision = "9c9de896864248929269a7cb2d64ed73b5fdf834"
+  revision = "6ed508ec6a4ecb3531899a69cbc746ccf65a4166"
```

Comment:
So, you are updating gorm too. Which makes sense (needed to align with auth) and a good thing regardless of auth update. Gorm has a bunch of important improvements/fixes in the updated version. But it may affect other code. I would talk to @sbryzak who made that original gorm upgrade in Auth to check if he remembers any additional changes we made in Auth along with that upgrade. Some default configuration may be changed or something... Just to be on the safe side here :)

Reply:
ok, let's wait for his feedback before merging the PR, then